pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -1052,6 +1059,55 @@ func pruneBundle(tx *gorm.DB, trustDomainID string, expiry time.Time, log logrus
 	return changed, nil
 }
 
+func taintJWTKey(tx *gorm.DB, trustDomainID string, taintedKeyID string) (*common.PublicKey, error) {
```

Comment:
Since the function _will_ taint the key, the taintedKeyID argument is confusing, because with that name I would expect to provide a key that is already tainted.

Maybe keyIDToTaint or just keyID are better options?

Reply:
I'll update to keyID